fixes.inc

```diff
@@ -8118,46 +8118,46 @@ static stock _FIXES_CheckMaxPlayers()
 				PlayerTextDrawTextSize__(playerid, t, 10.0, 200.0);
 
 				// Global style 13 (playerid, stunt).
-				t = FIXES_gsPGTStyle[playerid][13] = CreatePlayerTextDraw__(playerid, 380.000000, 341.150000, FIXES_gsSpace),
-				PlayerTextDrawLetterSize__(playerid, t, 0.579999, 2.420000),
-				PlayerTextDrawTextSize__(playerid, t, 40.000000, 460.000000),
+				t = FIXES_gsPGTStyle[playerid][13] = CreatePlayerTextDraw__(playerid, 380.0, 341.15, FIXES_gsSpace),
+				PlayerTextDrawLetterSize__(playerid, t, 0.58, 2.42),
 				PlayerTextDrawAlignment__(playerid, t, 2),
 				PlayerTextDrawColor__(playerid, t, 0xDDDDDBFF),
-				PlayerTextDrawUseBox__(playerid, t, true),
-				PlayerTextDrawBoxColor__(playerid, t, 0),
 				PlayerTextDrawSetShadow__(playerid, t, 2),
 				PlayerTextDrawSetOutline__(playerid, t, 0),
 				PlayerTextDrawBackgroundColor__(playerid, t, 170),
```

Comment:
Forgot about that, 0x000000AA would be more appropriate instead of 170, like in all other cases.